### ISSUE TITLE
self made scenario - building rights issue

### ISSUE THREAD

**Issue by DavydeVries** [OpenRCT2/OpenRCT2#3357]
**OS:** OS X El Capitan 10.11.5 (beta)
**Version:** Launcher 0.0.5 OpenRCT2 493 (0.0.5-developer)
**Commit:** ?

Make a scenario, make a entrance with build right behind the entrance. Guests can only walk one tile, not further.

- [ ] Reproducible in RCT2 (vanilla)? Unknown, can't play rtc2 on a mac....
- [ ] Multiplayer? no

**Steps to reproduce:**
1. Make an scenario
2. Build an entrance
3. Behind the entrance ground with build rights.
4. Save your scenario
5. Start/Play your scenario
6. Guests won't walk further than the first tile

**Dump file**
Nope..

**Screenshots / Video:**
<img width="760" alt="schermafbeelding 2016-04-20 om 17 31 35" src="https://cloud.githubusercontent.com/assets/7860305/14680494/c23a6dca-071d-11e6-95b4-b679c4d4e714.png">
<img width="549" alt="schermafbeelding 2016-04-20 om 17 36 40" src="https://cloud.githubusercontent.com/assets/7860305/14680749/9e8d1ac0-071e-11e6-94d9-33a05e14966d.png">

**Save game:**
Scenario file: [Buildrights issue.txt](https://github.com/OpenRCT2/OpenRCT2/files/228267/Buildrights.issue.txt) (old extension .sc6)

**Comment by IntelOrca** [comment]
I think this would probably also happen in vanilla RCT2. Construction rights don't allow construction directly on the ground.

**Comment by Gymnasiast** [comment]
I think it's the same issue #441. The gist of that one is a basically that construction rights placed using the window in the Scenario Editor render every path unusable, elevated or not. (Strangely, 8 cars seems to be capable of doing it correctly and the official Bumbly Beach conversion didn't suffer from it either.)

**Comment by IntelOrca** [comment]
@Gymnasiast no I think that issue is the scenario editor automatically setting any path outside the park as unowned, thus removing the construction rights.

**Comment by Gymnasiast** [comment]
Ah, ok.

@DavydeVries Could you lower the terrain under the path by at least two units, and check if the guests can use it?

**Comment by DavydeVries** [comment]
<img width="506" alt="schermafbeelding 2016-04-22 om 11 05 02" src="https://cloud.githubusercontent.com/assets/7860305/14736979/1c3e4dca-087a-11e6-8fc7-91bfd74a3cb8.png">
Sorry for my late answer. But nope. Or do you mean the whole path in de build rights area?

**Comment by IntelOrca** [comment]
Not quite, construction rights prohibit you from building on the ground - this its not supposed to work. But I expect we will have to change this anyway.

**Comment by DavydeVries** [comment]
I know that in vanilla you only can build stuff above the ground with build rights. In this case (see photo) the path is above the ground. ![](https://cloud.githubusercontent.com/assets/7860305/14737123/c328de02-087a-11e6-89f7-7e70bd037412.png)

The reason why I want to do this, creating a big plaza with build rights, so it can't be modified but coaster/monorail can be build above it.
